### DISCUSSION TITLE
What tools do you use?

### DISCUSSION THREAD

**Discussion by fade2gray** [mmikeww/AHK-v2-script-converter#228] [General]
Editor: VSCode
I don't think I could change this now that I've been using it for so long, with it's AHK v1 and v2 extensions.

Search/replace: dnGrep.
An excellent 'poor man's' version of PowerGREP. It has some limitations like the inability to use negative look-behinds because of its .Net regex flavour, but very useful all the same.  http://dngrep.github.io/

Local repo management: GitKraken (free)
I'm still getting used to it, but I like it. I had a look at GitUI, but I'm not a great fan of using command line utilities unless I absolutely need to.

**Comment by andymbody** [comment]
Editor:
 Scite. Don't really like the bloat of the larger IDEs, packages, especially MS tools

find/replace:
Notepad++, but not married to it.

Local git mgmt
SourceGit, really nice. I have very little experience with Git/Github. Still struggle with it, but this tool helps a lot.

**Comment by mmikeww** [comment]
editor: vim

git bash command line that ships with regular git for windows installation

**Comment by Banaanae** [comment]
IDE: VSCodium
Find Replace: [regex101](https://regex101.com)
Repo mgr: gitui for committing pushing, Git CLI for everything else

**Comment by CoffeeChaton** [comment]
Editor: VSCode
Search/replace: VSCode.
git tools: GitHub Desktop / Fork.exe / vscode
